### PR TITLE
Add Order "C" and "D" of CRUD.

### PR DESCRIPTION
**Pull request by Jaspertzx**
Currently, there are no methods or classes to
create the Order object.

This prevent the User from doing the interacting
with the order objects.

Implement the read and delete of Order object.

This will allow the user to interact with the
created objects.

### CODE REVIEW

**Comment by Jaspertzx** [comment]
Resolves issue #50

**Review by kailinteoo — APPROVED** [review]
LGTM! Please make more frequent PR instead of committing 38 different files in one go, but overall very good job! thanksss!

File: src/main/java/seedu/address/logic/parser/DeleteOrderCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/**
+ * Parses input arguments and creates a new DeleteOrderCommand object
+ */
+public class DeleteOrderCommandParser implements Parser<DeleteOrderCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the
+     * DeleteOrderCommand and returns a DeleteOrderCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```
Comment (rertyy) [inline src/main/java/seedu/address/logic/parser/DeleteOrderCommandParser.java:21]: Missing ending full stop

File: src/main/java/seedu/address/model/Model.java [hunk 2]
```diff
@@ -84,4 +87,19 @@ public interface Model {
...
+     */
+    void addOrder(Order order) throws CommandException;
+
+    /**
+     * Deletes the given Order.
+     * The order must exist in the order list.
+     */
+    void deleteOrder(Order order) throws CommandException;
+
+    /** Returns an unmodifiable view of the order list */
```
Comment (rertyy) [inline src/main/java/seedu/address/model/Model.java:102]: Missing ending full stop

File: src/test/java/seedu/address/logic/commands/AddOrderCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+    @Test
+    public void execute_orderAcceptedByModel_addSuccessful() throws Exception {
+        OrderBuilder builder = new OrderBuilder();
+        Order order = builder.build();
+        ModelStubAcceptingOrderAdded modelStub = new ModelStubAcceptingOrderAdded(order);
+        CommandResult commandResult = new AddOrderCommand(order).execute(modelStub);
+        assertEquals(String.format(MESSAGE_SUCCESS, Messages.format(order)),
+                commandResult.getFeedbackToUser());
+    }
```
Comment (rertyy) [inline src/test/java/seedu/address/logic/commands/AddOrderCommandTest.java:32]: Can be extracted to separate variable

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 4]
```diff
@@ -88,6 +92,23 @@ public void parseCommand_list() throws Exception {
...
+        assertTrue(parser.parseCommand(AddOrderCommand.COMMAND_WORD
+                + remarks + cost + date) instanceof AddOrderCommand);
+    }
+
+    @Test
+    public void parseCommand_deleteOrder() throws Exception {
+        OrderBuilder builder = new OrderBuilder();
+        OrderId orderId = builder.build().getOrderId();
+        assertTrue(parser.parseCommand(DeleteOrderCommand.COMMAND_WORD
+                + " id/" + orderId.toString()) instanceof DeleteOrderCommand);
```
Comment (rertyy) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:109]: Likewise, too long

**Review by rertyy — COMMENTED** [review]
LGTM, small code style nits

**Pull request merged**